source/material_model/utilities.cc

```diff
@@ -786,6 +788,91 @@ namespace aspect
           AssertThrow(i == n_temperature*n_pressure, ExcMessage("Material table size not consistent with header."));
 
         }
+
+
+
+        void
+        EntropyReader::initialize(const MPI_Comm comm,
+                                  const std::string data_directory,
+                                  const std::string material_file_name)
+        {
+          material_lookup = std::make_unique<Utilities::StructuredDataLookup<2>>(7,1.0);
+          material_lookup->load_file(data_directory+material_file_name,
+                                     comm);
+        }
+
+        double
+        EntropyReader::specific_heat(const double entropy,
+                                     const double pressure) const
+        {
+          Point<2> entropy_pressure(entropy, pressure);
+          const double specific_heat = material_lookup->get_data(entropy_pressure, 3);
```

Comment:
I know you moved this from another place, but we now have a better way to write this:

```suggestion
          const double specific_heat = material_lookup->get_data({entropy,pressure}, 3);
```

This feature is called an initializer_list and used to create the Point implicitly, so we save a line of code (and the point is only available inside of the `get_data` function.
You can change this for all places below as well.